images/base/Dockerfile

```diff
@@ -101,7 +100,7 @@ RUN echo "Ensuring scripts are executable ..." \
     && rm -rf /tmp/crictl.tgz \
  && echo "Installing CNI binaries ..." \
     && export CNI_TARBALL="${CNI_VERSION}/cni-plugins-linux-${ARCH}-${CNI_VERSION}.tgz" \
-    && export CNI_URL="https://github.com/bentheelder/containernetworking-plugins/releases/download/${CNI_TARBALL}" \
```

Comment:
I don't know how I did build this locally :face_with_head_bandage: , but it created the container